src/main/java/frc/robot/RobotContainer.java

```diff
@@ -93,7 +93,8 @@ public RobotContainer() {
    */
   public void resetToAbsolutePositions() {
     subDrivetrain.resetSteerMotorEncodersToAbsolute();
-    subWrist.resetEncoderToAbsolute();
+    subWrist.resetWristEncoderToAbsolute();
+    subElevator.resetElevatorEncoderToAbsolute();
```

Comment:
Do we want to move this out of this method to allow this method to be called periodically